### PR TITLE
Remove partner token support [#168661200]

### PR DESCRIPTION
**Pull request by vitaliypodlypnyi**

### CODE REVIEW

File: CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs [hunk 1]
```diff
@@ -54,36 +52,33 @@ public async Task OnGet()
...
-                    user.CompanyName, 
-                {userSecretKey}, 
-                {userStoreId});
+                var connectionId = await CreateConnection(user.CompanyName, {userSecretKey}, {userStoreId}, otc);
                 
                 user.ZgConnectionId = connectionId;
                 await _userManager.UpdateAsync(user);
             }
-
+
```
Comment (dennisahlin) [inline CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:67]:
```suggestion

```

File: CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs [hunk 2]
```diff
@@ -54,36 +52,33 @@ public async Task OnGet()
             //Fetch user
             var user = await _userManager.GetUserAsync(User);
             
+            //Receive one time code, using client id and secret
```
Comment (dennisahlin) [inline CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:55]:
```suggestion
            // Fetch one time code, using client id and secret
```

File: CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs [hunk 3]
```diff
@@ -244,7 +239,7 @@ internal class OneTimeCodeRequest
     
     internal class OneTimeCodeResponse
     {
-        [JsonProperty("one_time_code")]
+        [JsonProperty("oneTimeCode")]
```
Comment (dennisahlin) [inline CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:242]: Why is JsonProperty different? Why not name the property `OneTimeCode`?

Comment (vitaliypodlypnyi) [reply CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs]: fixed

**Pull request merged**